trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapCoverEndDate.js

```diff
@@ -3,6 +3,10 @@ const { hasValue } = require('../../../../utils/string');
 const { formatYear } = require('../../../../utils/date');
 
 const mapCoverEndDate = (day, month, year) => {
+  // checks if valid date - not valid if coverEndDate is not set
+  if (Number.isNaN(Number(day)) || Number.isNaN(Number(month)) || Number.isNaN(Number(year))) {
```

Comment:
Can we use `Number.isInteger` or `Number.isSpecialInteger` please above looks like a gotcha.

Reply:
removed